imap_processing/mag/l1a/mag_l1a_data.py

```diff
@@ -47,6 +46,7 @@ def __add__(self, seconds: float):  # type: ignore[no-untyped-def]
         -------
         time : TimeTuple
             New time tuple with the current time tuple + seconds.
+            Ruff is saying TimeTuple is undefined for this usage.
```

Comment:
This doesn't really belong in the docstring for the return type.

I'm not sure what it is trying to say, but can you remove the `ignore` by specifying the explicit return type to the function definition `-> TimeTuple`